src/utils/editor/index.js

```diff
@@ -34,8 +35,12 @@ function shouldShowFooter(selectedSource, horizontal) {
   if (!horizontal) {
     return true;
   }
-
-  return shouldShowPrettyPrint(selectedSource);
+  if (!selectedSource) {
+    return false;
+  }
+  return (
+    shouldShowPrettyPrint(selectedSource) || isOriginal(selectedSource.toJS())
```

Comment:
why can't we use `isOriginalId(selectedSource.get("id"))` instead of `isOriginal(selectedSource.toJS())`here.

Reply:
turned out both give same results, updated to `isOriginalId(selectedSource.get("id"))`